doc/content/lorawan/eirp-and-erp/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "EIRP and ERP"
+section: Additional Information
+description: ""
+weight:
+---
+
+LoRaWAN gateways and end devices consist of radios (also called transceivers) that are capable of transmitting and receiving LoRa modulated radio signals. A radio needs an antenna to transmit or receive signals. The connection between the radio and the antenna is usually made by a piece of cable (for some devices it is a PCB trace). When transmitting, radio (transmitter) and antenna provide signal gain while antenna cable and connectors affect signal loss.
+
+## EIRP
+
+EIRP (Effective Isotropic Radiated Power) is the total power radiated by an isotropic antenna in a single direction. In this case, the antenna gain must be expressed in dBi.
+
+{{< figure src="eirp.png" alt="EIRP" >}}
+
+The EIRP of a unit that consists of a transmitter, an antenna, and a cable can be calculated using the formula,
+
+**EIRP = Transmitter output power (dBm) + Antenna gain (dBi) - Cable loss (dB)**
+
+where the transmitter power in dBm, antenna gain in dBi, and cable loss in dB.
+
+The maximum EIRP of a gateway or an end device is mentioned for various regions by the respective regulatory authorities. For example, in Europe, it is +16 dBm and regulated by ETSI (European Telecommunications Standards Institute), in America it is +30 dBm and regulated by FCC (The Federal Communications Commission), and in Australia, it is +30 dBm and regulated by ACMA (Australian Communications and Media Authority). Read the [LoRaWAN® Regional Parameters 1.0.3](https://lora-alliance.org/wp-content/uploads/2021/05/RP002-1.0.3-FINAL-1.pdf) to find the maximum allowed EIRP in other regions and their respective regulatory authorities. 
```

Comment:
> For example, in Europe, it is +16 dBm and regulated by ETSI (European Telecommunications Standards Institute)

Looking at [the regulations](https://www.etsi.org/deliver/etsi_en/300200_300299/30022002/03.02.01_60/en_30022002v030201p.pdf) it is actually specified as E.R.P. and in milliwatt.

Reply:
I have edited the paragraph and moved it to a new section.